app/models/mixins/owner_mixin.rb

```diff
@@ -0,0 +1,13 @@
+module OwnerMixin
+  extend ActiveSupport::Concern
+
+  included do
+    validates :owner, :presence => true, :on => :create
+
+    before_validation :set_owner, :if => proc { ManageIQ::API::Common::Request.current.present? }, :on => :create
```

Comment:
Is `proc` valid syntax?  I thought it had to be `Proc.new`

Reply:
Yeah, for some reason my rubocop was complaining about it, it was saying use `proc` instead of `Proc.new`, I'll change it and see if our rubocop complains